workos/__about__.py

```diff
@@ -12,7 +12,7 @@
 
 __package_url__ = "https://github.com/workos-inc/workos-python"
 
-__version__ = "0.7.0"
+__version__ = "0.8.0"
```

Comment:
Can we make this 0.7.1?

Reply:
@rjvani I was thinking it's a minor bump since [it's a backwards-compatible feature addition instead of a fix](https://github.com/workos-inc/workos/wiki/SDK-Release-Process#versioning-policy).